starboard/android/apk/app/src/main/java/dev/cobalt/media/VideoSurfaceView.java

```diff
@@ -75,6 +78,19 @@ private void initialize(Context context) {
     // punch-out video when the position / size is animated.
   }
 
+  public void clearSurface() {
+    if (getHolder().getSurface().isValid()) {
+      Canvas canvas = getHolder().lockCanvas();
+      if (canvas != null) {
+        canvas.drawColor(Color.TRANSPARENT, PorterDuff.Mode.CLEAR);
```

Comment:
The old code draws black instead of transparent, but I don't know if it would make any difference.

Reply:
Right. Black works just as well. Updated.